src/index.js

```diff
@@ -0,0 +1,62 @@
+import _ from 'lodash';
+import './style.css';

```

Comment:
Please also try to import your CSS files before node modules for easy readability

Reply:
Done 👍